src/components/sideBarContentEdit/SelectDirectory.tsx

```diff
@@ -220,7 +223,7 @@ const SelectDirectory = ({
             }}
             height={8}
             onChange={handleChange}
-            placeholder={path}
+            placeholder={!isOpen ? updatedTranscript?.loc : path}
```

Comment:
It seems that the logic introduced with 069bef7795a089f3e7738d4f6ecfe36064ae0707 solves just one of the issues listed in #186. Resetting directory (`loc`) still doesn't work, as it still resets to the current saved state (`content`) and not `originalContent`. It seems that this behavior was wrongly introduced with this line: 
https://github.com/bitcointranscripts/transcription-review-front-end/blob/092ad3848e7ce344a2c076957129208ea9710371/src/components/transcript/index.tsx#L133-L135